tests/test_latest.py

```diff
@@ -283,7 +284,7 @@ def test_multiscale_group_datasets_exist(default_multiscale: Multiscale):
     ]
     MultiscaleGroup(
         name="",
-        attrs={"multiscales": [default_multiscale.dict()]},
+        attrs=MultiscaleAttrs(multiscales= [default_multiscale]),
```

Comment:
I think v2 has got less flexible about taking instances which are a mixture of structured models and dicts. These tests did pass under v1 not but under v2 unless the whole tree is as models.